app/cdap/components/PipelineDiff/DiffIcon.tsx

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright © 2023 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+import React from 'react';
+import EditIcon from '@material-ui/icons/Edit';
+import AddBoxOutlinedIcon from '@material-ui/icons/AddBoxOutlined';
+import IndeterminateCheckBoxOutlinedIcon from '@material-ui/icons/IndeterminateCheckBoxOutlined';
+import { DiffIndicator } from './types';
+import { getPluginDiffColors } from './util/helpers';
+
+interface IDiffIconProps {
+  diffType: DiffIndicator;
+  fontSize?: 'default' | 'inherit' | 'large' | 'medium' | 'small';
```

Comment:
can this be default?

Reply:
[Yes](https://v4.mui.com/api/svg-icon/#svgicon-api). But I adjusted the component so it acts like a `SvgIcon` wrapper.